ArduCopter/Log.cpp

```diff
@@ -472,6 +481,15 @@ const struct LogStructure Copter::log_structure[] = {
 
     { LOG_CONTROL_TUNING_MSG, sizeof(log_Control_Tuning),
       "CTUN", "Qffffffefffhh", "TimeUS,ThI,ABst,ThO,ThH,DAlt,Alt,BAlt,DSAlt,SAlt,TAlt,DCRt,CRt", "s----mmmmmmnn", "F----00B000BB" },
+    
+// @LoggerMessage: MOTB
+// @Description: Battery information
+// @Field: TimeUS: Microseconds since system startup
+// @Field: LiftMax: Maximum lift ratio
```

Comment:
An explanation of this would be good.  The code is remarkably opaque on what this is.  I think it might be "maximum motor compensation gain"?